web-app/js/portal/prototypes/OpenLayers.js

```diff
@@ -151,20 +151,53 @@ OpenLayers.Layer.WMS.prototype.isNcwms = function() {
     return false;
 };
 
+OpenLayers.Layer.WMS.prototype.isAla = function() {
+    return false;
+};
+
 OpenLayers.Layer.WMS.prototype.applyFilters = function(filters) {
 
-    var builder = new Portal.filter.combiner.MapCqlBuilder({
+    if (this.isAla()) {
+        this.applyAlaFilters(filters);
+    }
+    else {
+
+        var builder = new Portal.filter.combiner.MapCqlBuilder({
+            filters: filters
+        });
+
+        var newValue = builder.buildCql();
+        var existingValue = this.params['CQL_FILTER'];
+
+        if (newValue != existingValue) {
+            this.mergeNewParams({
+                CQL_FILTER: newValue
+            });
+        }
+    }
+};
+
+OpenLayers.Layer.WMS.prototype.applyAlaFilters = function(filters) {
```

Comment:
Can this just be included in AlaWms below rather than making it generally available on all layers?

Reply:
yes. todo then!